packages/caver-rpc/src/governance.js

```diff
@@ -149,8 +149,8 @@ const Governance = function Governance(...args) {
             params: 0,
         }),
         /**
-         * Provides the initial chain configuration.
-         * Because it just stores the initial configuration, if there were changes in the governance made by voting, the result of chainConfig will differ from the current states.
+         * Provides the latest chain configuration.
+         * This is equivalent to chainConfigAt() with an empty parameter.
          * To see the current information, please use {@link itemsAt}.
```

Comment:
Don't you need to delete this? `getChainConfig` itself will return latest chain configuration.

Reply:
Good point. 
It seems that the comments in the previous document were verbose.